build.gradle

```diff
@@ -138,6 +138,10 @@ configure(javaProjects) {
             dependency("org.glassfish.jersey.containers:jersey-container-servlet:2.19")
             dependency("org.glassfish.jersey.media:jersey-media-json-jackson:2.19")
             dependency("org.spockframework:spock-guice:1.0-groovy-2.4")
+            /**es 5.4.1 dependencies*/
+            dependency("org.elasticsearch.client:transport:5.4.1")
+            dependency("org.apache.logging.log4j:log4j-api:2.7")
```

Comment:
Why do we need these explicitly here?

Reply:
To pin the version of elasticSearch client instead of relying on the default one.  I removed the dependency of org.apache.logging.log4j:log4j-api:2.7.